Tests/StackClusterTests.swift

```diff
@@ -161,7 +163,7 @@ class StackClusterTests: QuickSpec {
                     it("should be able to scroll horizontally") {
                         let vc = nestedStack()
                         vc?.loadViewIfNeeded()
-                        
+                        vc?.view.layoutIfNeeded()
```

Comment:
@ChristianOrgler is right. @joanromano @MP0w you don't need to load the view to take snapshots, that's true. But you have three expectations: `expect(vc).to(haveValidSnapshot())
                    expect(scrollView).to(scroll(.horizontal))
                    expect(scrollView).toNot(scroll(.vertical))`
In my opinion these expectations should be independent and the order should not matter. Having to check haveValidSnapshot() should not be a precondition for the other two expectations. 

> you don't need to check here if it can scroll

I did not add these, I just fixed that they will succeed, even without checking for a snapshot.